### DISCUSSION TITLE
Should i enable ipv6?, FF Rules ok? / MagentaTV

### DISCUSSION THREAD

**Discussion by Trackhe** [fabianishere/udm-iptv#74] [Q&A]
Actually i have disabled ipv6 on the iptv subnetwork, but should i enable it? and what are the correct ff rules? 
Actually i use 

Accept IGMP Internet In 
Source: MagentaIPTVWAN:Any
Dest: Any:Any

Accept UDP Internet In 
Source: MagentaIPTVWAN:Any
Dest: Any:Any

Accept IGMP Internet Local
Source: Any:Any
Dest: Any:Any

MagentaIPTVWAN: IPV4: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15

It seems to work the most of the time, sometimes the tv shows a relly bad quallity. Like big pixels.
And i m not sure that configuration is secure and correct.

**Comment by fabianishere** [comment]
I believe the IPTV traffic is carried over IPv4. Why don’t you try enabling it?
